client/src/app/pages/assessment/components/assessment-wizard/assessment-wizard.tsx

```diff
@@ -506,62 +503,6 @@ export const AssessmentWizard: React.FC<AssessmentWizardProps> = ({
     }
   };
 
-  const wizardSteps: WizardStep[] = [
-    {
-      id: 0,
-      name: t("composed.selectMany", {
-        what: t("terms.stakeholders").toLowerCase(),
-      }),
-      component: <AssessmentStakeholdersForm />,
-      canJumpTo: 0 === currentStep || !disableNavigation,
-      enableNext: isFirstStepValid(),
-    },
-    ...sortedSections.map((section, index) => {
-      const stepIndex = index + 1;
-
-      return {
-        id: stepIndex,
-        name: section.name,
-        stepNavItemProps: {
-          children: <WizardStepNavDescription section={section} />,
-        },
-        component: <QuestionnaireForm key={section.name} section={section} />,
-        canJumpTo:
-          stepIndex === currentStep ||
-          (stepIndex <= canJumpTo && !disableNavigation),
-        enableNext: shouldNextBtnBeEnabled(section),
-      } as WizardStep;
-    }),
-  ];
-
-  const wizardFooter = (
-    <CustomWizardFooter
-      isFirstStep={currentStep === 0}
-      isLastStep={currentStep === sortedSections.length}
-      isDisabled={
-        isSubmitting ||
-        isValidating ||
-        (currentStep === sortedSections.length &&
-          !shouldNextBtnBeEnabled(sortedSections[currentStep - 1]))
-      }
-      hasAnswers={hasPartialAnswers(sortedSections[currentStep - 1])}
-      isFormInvalid={!isValid}
-      onSave={(review) => {
-        const saveActionValue = review
-          ? SAVE_ACTION_VALUE.SAVE_AND_REVIEW
-          : SAVE_ACTION_VALUE.SAVE;
-
-        methods.setValue(SAVE_ACTION_KEY, saveActionValue);
-
-        methods.handleSubmit(onSubmit, onInvalid)();
-      }}
-      onSaveAsDraft={() => {
-        methods.setValue(SAVE_ACTION_KEY, SAVE_ACTION_VALUE.SAVE_AS_DRAFT);
-        methods.handleSubmit(onSubmit)();
-      }}
-    />
-  );
-
   useEffect(() => {
     const unlisten = history.listen((newLocation, action) => {
       if (action === "PUSH" && assessment) {
```

Comment:
Looks like something with the way the new wizard works is causing this to trigger on next button click. Need to figure out how to clear the assessment when the user navigates away from the wizard without explicitly saving/cancelling the wizard so that we don't end up with an orphaned assessment.